src/azure-cli/azure/cli/command_modules/netappfiles/_help.py

```diff
@@ -362,6 +370,93 @@
+parameters:
+  - name: --account-name -a
+    short-summary: The name of the ANF account
+  - name: --pool-name -p
+    short-summary: The name of the ANF pool
+  - name: --name --volume-name -n -v
+    short-summary: The name of the replication source volume
+  - name: --remote-volume-resource-id -d
+    short-summary: The resource id of the destination replication volume
+examples:
+  - name: Authorize the volume as the replication destination for the source
+    text: >
+        az netappfiles volume replication approve -g mygroup --account-name myaccname --pool-name mypoolname --name mysourcevolname --remote-volume-resource-id /subscriptions/69a75bda-882e-44d5-8431-63421204131c/resourceGroups/mygroup1/providers/Microsoft.NetApp/netAppAccounts/myaccount1/capacityPools/mypool1/volumes/mydestinationvolume
+"""
+
+helps['netappfiles volume replication suspend'] = """
+type: command
+short-summary: Suspend/break a volume replication for the specified destination volume. The replication process is suspended until resumed or deleted.
+parameters:
+  - name: --account-name -a
+    short-summary: The name of the ANF account
+  - name: --pool-name -p
+    short-summary: The name of the ANF pool
+  - name: --name --volume-name -n -v
+    short-summary: The name of the replication destination volume
+examples:
+  - name: Suspend the replication process
+    text: >
+        az netappfiles volume replication suspend -g mygroup --account-name myaccname --pool-name mypoolname --name mydestinationvolname
+"""
+
+helps['netappfiles volume replication resume'] = """
+type: command
+short-summary: Resync a volume replication for the specified destination volume. The replication process is resumed from source to destination.
+parameters:
+  - name: --account-name -a
+    short-summary: The name of the ANF account
+  - name: --pool-name -p
+    short-summary: The name of the ANF pool
+  - name: --name --volume-name -n -v
+    short-summary: The name of the replication destination volume
+examples:
+  - name: Resume the replication process
+    text: >
+        az netappfiles volume replication resume -g mygroup --account-name myaccname --pool-name mypoolname --name mydestinationvolname
+"""
+
+helps['netappfiles volume replication remove'] = """
+type: command
+short-summary: Delete a volume replication for the specified destination volume. The data replication objects of source and destination volumes will be removed.
+parameters:
+  - name: --account-name -a
+    short-summary: The name of the ANF account
+  - name: --pool-name -p
+    short-summary: The name of the ANF pool
+  - name: --name --volume-name -n -v
+    short-summary: The name of the replication destination volume
+examples:
+  - name: Delete the replication objects of the paired volumes
+    text: >
+        az netappfiles volume replication remove -g mygroup --account-name myaccname --pool-name mypoolname --name mydestinationvolname
+"""
+
+helps['netappfiles volume replication status'] = """
```

Comment:
What about changing to show command?

Reply:
I suggest we do this change for next version/update that will come very soon.